src/auth/auth-manager.js

```diff
@@ -118,37 +119,23 @@ const verify = (req, res, next) => {
   const token = req.body.token || req.query.token || req.headers['x-access-token'];
 
   // If no token is provided, inform the client.
-  if (!token) {
-    next({ err: 'No token provided.' });
-    return;
-  }
+  if (!token) { throw new Error('No token provided.'); }
```

Comment:
Wouldn't throwing an exception just crash the app/go to a separate error handling function (not the endpoint using the middleware)?  If we can't pass errors directly to next, and endpoints using this middleware still need to see these errors and handle them appropriately, having an ```err``` object as part of ```res.locals.auth``` might be the way to go on this.